### ISSUE TITLE
Bug: Contrast too low for PPC election form title

### ISSUE THREAD

**Issue by hugovk** [python/pythondotorg#3070]
### Describe the bug

The contrast ratio for the "Submit a Nomination for 2026 Python Packaging Council Election" title is too low:

<img width="1199" height="383" alt="Image" src="https://github.com/user-attachments/assets/bc4294df-78af-4d6f-9560-f476712c2f55" />

https://www.python.org/nominations/2026-python-packaging-council/create/

It's 1.44, and should be at least 3.0 to meet WCAG AA accessibility requirements:

<img width="233" height="83" alt="Image" src="https://github.com/user-attachments/assets/22c376c6-1970-4891-8c48-3af91bdac401" />

The colour is defined in the database 

I suggest using a darker green, like `#2E7D32` with a ratio of 4.99:

<img width="1194" height="360" alt="Image" src="https://github.com/user-attachments/assets/ad65984b-f634-4e55-bad6-d159364b8c6c" />

<img width="236" height="87" alt="Image" src="https://github.com/user-attachments/assets/ebd965ad-ffdf-420f-a489-8fce5b1d0b95" />

To change it, someone with access to nominations should do something like:

Admin > Nominations > [Election kinds](https://www.python.org/admin/nominations/electionkind/) > Packaging Council > change "Accent color" to #2E7D32.

### Browsers

Chrome

### Operating System

macOS

### Additional context

(The contrast is good of the blue for the PSF board election at https://www.python.org/nominations/2026-python-software-foundation-board/create/)

<img width="1205" height="457" alt="Image" src="https://github.com/user-attachments/assets/c208fc87-ddb9-4af6-ab0e-ff578179da16" />

**Comment by yash-gupta-code-dev** [comment]
How can I fix it is it allowed? Please let me know if I can fix some of the issue and if yes then how is it possible

**Comment by hugovk** [comment]
@yash-gupta-code-dev 

> To change it, someone with access to nominations should do something like:
> 
> Admin > Nominations > [Election kinds](https://www.python.org/admin/nominations/electionkind/) > Packaging Council > change "Accent color" to #2E7D32.

**Comment by JacobCoffee** [comment]
Staff will fix in admin.
